### PR TITLE
Create pull request template for the frontend

### LINKED ISSUE #233: Create a Pull Request template for both the frontend and backend repos
This should have sections for:
* What Issue it is solving
* ~How to manage labels/assignments/various PR settings~
* ~Reminders of what to test for before putting up a PR~
* Any special requirements to test, any helpful knowledge for the reviewer
* Other acceptance criteria for code like
  * Code is readable and formatted
  * There isn't any unnecessary commented-out code

### PR DESCRIPTION
**Pull request by Chris-Boe**
### What issue is this solving?
Closing Issue #233 

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? Nope!
Any special requirements to test? No testing required

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### What issue is this solving?
+Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.
+
+### Any helpful knowledge/context for the reviewer?
```
Comment (dbridenbeck) [inline pull_request_template.md:4]: I like this structure! I go back and forth on if we should make the heading here a bit more pointed to help get more specific feedback . . . but then I also worry that asking an overly specific question might not fit all cases and could confuse folks.

My hunch is to keep it simple like this for now and later we can get more specific if we feel we need to.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Is a `npm install` necessary?
+Any special requirements to test?
+
+### Please make sure you've attempted to meet the following coding standards
+- [ ] Code builds successfully
+- [ ] Code has been tested and does not produce errors
+- [ ] Code is readable and formatted
+- [ ] There isn't any unnecessary commented-out code
+- [ ] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)
```
Comment (dbridenbeck) [inline pull_request_template.md:13]: After this checklist we might want to put a friendly disclaimer like, "If you're having trouble meeting this criteria reach out on slack and we'll help!"

**Review by dbridenbeck — DISMISSED** [review]
I think this looks great. I shared a few thoughts to consider but don't feel my suggestions are necessary if you have strong feelings one way or another :)

File: pull_request_template.md (hunk 2 above)
Comment (Chris-Boe) [reply pull_request_template.md:13]: I like that! I added it :)

**Pull request merged**